cristin-person/src/main/java/no/unit/nva/cristin/person/affiliations/handler/PositionCodesHandler.java

```diff
@@ -35,11 +35,16 @@ public PositionCodesHandler(CristinPositionCodesClient apiClient, Environment en
     protected PositionCodes processInput(Void input, RequestInfo requestInfo, Context context)
         throws ApiGatewayException {
 
-        return apiClient.generateQueryResponse();
+        return apiClient.generateQueryResponse(extractPositionCodeStatusLookingFor(requestInfo));
     }
 
     @Override
     protected Integer getSuccessStatusCode(Void input, PositionCodes output) {
         return HttpURLConnection.HTTP_OK;
     }
+
+    private Boolean extractPositionCodeStatusLookingFor(RequestInfo requestInfo) {
+        return requestInfo.getQueryParameterOpt(ACTIVE_STATUS_QUERY_PARAM)
+            .map(Boolean::valueOf).orElse(null);
```

Comment:
Does this work if the value is something else than boolean? What if i add param `active=hello` ?